plugins/destination/snowflake/client/read.go

```diff
@@ -12,61 +13,85 @@ const (
 	readSQL = "SELECT * FROM %s WHERE \"_cq_source_name\" = ?"
```

Comment:
Should this also be `order by _cq_sync_time asc` like the other plugins?

Reply:
I think it is like that be default